docs/docs/getting-started/installation.md

```diff
@@ -6,6 +6,15 @@ sidebar_position: 2
 
 ```mdx-code-block
 import TerminalWindow from '@site/src/components/TerminalWindow';
+import DocumentMetadata from '@site/src/components/DocumentMetadata';
+```
+
+```mdx-code-block
+<DocumentMetadata
+    lastUpdatedAt="{{last-updated-at}}"
+    authorInfo="{{author-info}}"
```

Comment:
Why the author info :thinking: ?

Reply:
no real good answer for this.
removing 🏃